Frameworks/CoreText/CTFont.mm

```diff
@@ -101,25 +110,74 @@
 const CFStringRef kCTFontFeatureSelectorSettingKey = static_cast<CFStringRef>(@"kCTFontFeatureSelectorSettingKey");
 
 /**
+ All these values are fixed number in IOS no matter on iphone* or ipad*.
+ */
+const CGFloat kCTFontLabelFontSize = 17.0f;
+const CGFloat kCTFontButtonFontSize = 14.0f;
+const CGFloat kCTFontSmallSystemFontSize = 12.0f;
+const CGFloat kCTFontSystemFontSize = 14.0f;
+
+using namespace Microsoft::WRL;
+
+struct __CTFont {
+    CFRuntimeBase _base;
```

Comment:
Thanks but this is mostly just a copy of CF structs... ^^;;

---
In reply to: [78219486](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78219486) [](ancestors = 78219486)